package.json

```diff
@@ -29,6 +29,13 @@
     "url": "https://github.com/epam/deps-fe-usePolling/issues"
```

Comment:
i don't know how to mention unchanged lines. but it's required to update package version, before publishing

Reply:
you can do it manually, or via cli `npm version <patch | minor | major>`

https://docs.npmjs.com/cli/v9/commands/npm-version

Reply:
I prefer to set it manually and review in code